musicbot/bot.py

```diff
@@ -2449,15 +2461,35 @@ def check(message):
         await self.disconnect_all_voice_clients()
         raise exceptions.RestartSignal()
 
-    async def cmd_shutdown(self, channel):
-        await self.safe_send_message(channel, "\N{WAVING HAND SIGN}")
-        
-        player = self.get_player_in(channel.guild)
-        if player and player.is_paused:
-            player.resume()
-        
-        await self.disconnect_all_voice_clients()
-        raise exceptions.TerminateSignal()
+    # This uses a lot of the search command selection functionality
+    async def cmd_shutdown(self, channel, message):
```

Comment:
Can you please update to use discord.py rewrite syntax? https://discordpy.readthedocs.io/en/rewrite/migrating.html